test/e2e/psp.feature.ts

```diff
@@ -40,19 +43,20 @@ const readBeaconValue = async (airnodeAddress: string, templateId: string, dapiS
   const beaconId = hre.ethers.utils.keccak256(
     hre.ethers.utils.solidityPack(['address', 'bytes32'], [airnodeAddress, templateId])
   );
-  const dapiServerResponse = await dapiServer.connect(voidSigner).readWithDataPointId(beaconId);
 
-  return dapiServerResponse[0].toNumber();
+  try {
+    const dapiServerResponse = await dapiServer.connect(voidSigner).readDataFeedValueWithId(beaconId);
+    return dapiServerResponse.toNumber();
+  } catch (e) {
+    return null;
+  }
```

Comment:
The function name has changed to `readDataFeedValueWithId` and it returns only the dapiServer value instead of an array: `[value, timestamp]`.

The new method also has added a check that the **dataFeed** exists instead of returning [0, timestamp] like the earlier version, hence the **try-catch**.